### ISSUE TITLE
1.20.1

### ISSUE THREAD

**Issue by ZmogusTapke007** [19MisterX98/SeedcrackerX#234]
Please make irt for 1.20.1 i cantcuse it in new servers

**Comment by JACKKY04** [comment]
Any update on this? 1.20.1 availability.

**Comment by Pommyd** [comment]
Yes, you can download the [prerelease](https://github.com/19MisterX98/SeedcrackerX/releases/tag/prerelease).
